pt/development/sessions.rst

```diff
@@ -1,17 +1,424 @@
+sessão 'php'. As configurações internas são:
+
+* ``php`` - Salva sessões com as configurações padrão no seu arquivo php.ini.
+* ``cake`` - Salva sessões como arquivos dentro de ``tmp/sessions``. Essa é uma boa opção quando 
+  em hosts que não permitem que você escreva fora de seu próprio diretório. 
+* ``database`` - Use as sessões de banco de dados internas. Veja abaixo para mais informações.
+* ``cache`` - Use as sessões de cache internas. Veja abaixo para mais informações.
+
+Manipuladores de Sessão
+-----------------------
+
+Session handlers can also be defined in the session config array.  By defining
+the 'handler.engine' config key, you can name the class name, or provide
+a handler instance.  The class/object must implement the
+native PHP ``SessionHandlerInterface``. Implementing this interface will allow
+``Session`` to automatically map the methods for the handler. Both the core
+Cache and Database session handlers use this method for saving sessions.
+Additional settings for the handler should be placed inside the handler array.
+You can then read those values out from inside your handler::
+
+Os manipuladores de sessão também podem ser definidos na matriz de configuração 
+da sessão. Ao definir a chave de configuração 'handler.engine', você pode nomear 
+o nome da classe ou fornecer uma instância do manipulador. A classe/objeto deve 
+implementar o PHP nativo ``SessionHandlerInterface``. A implementação dessa 
+interface permitirá que a ``Session`` mapeie automaticamente os métodos para 
+o manipulador. Os principais manipuladores de sessão do Cache e do Banco de 
+Dados usam esse método para salvar sessões. Configurações adicionais para o manipulador 
+devem ser colocadas dentro da matriz do manipulador. Você pode então ler esses valores 
+de dentro do seu manipulador::
+
+    'Session' => [
+        'handler' => [
+            'engine' => 'DatabaseSession',
+            'model' => 'CustomSessions'
+        ]
+    ]
+
+A tabela acima mostra como você pode configurar o manipulador de sessões do banco de 
+dados com um modelo de aplicativo. Ao usar nomes de classe como seu handler.engine, 
+o CakePHP espera encontrar sua classe no espaço de nome ``Http\Session``. Por exemplo, 
+se você tiver uma classe ``AppSessionHandler``, o arquivo deve ser 
+**src/Http/Session/AppSessionHandler.php** e o nome da classe deve ser ``App\Http\Session\AppSessionHandler``. 
+Você também pode usar manipuladores de sessão de plugins internos. Configurando o 
+mecanismo para ``MyPlugin.PluginSessionHandler``.
+
 .. note::
-    Atualmente, a documentação desta página não é suportada em português.
+    Antes da versão 3.6.0, os arquivos do adaptador de sessão devem ser colocados em
+    **src/Network/Session/AppHandler.php**.
+
+
+Sessões de Banco de Dados
+-------------------------
+
+Se você precisar usar um banco de dados para armazenar os dados da sessão, configure da seguinte maneira:
```

Comment:
Need a second : at the end of this line.

Reply:
Ops! I'ts not ready